main.go

```diff
@@ -83,10 +84,16 @@ func main() {
 				EnvVars: []string{"JAEGER_SAMPLER_PARAM"},
 				Value:   0.0001,
 			},
+			&cli.BoolFlag{
+				Name:    "allow-schema-migration",
+				EnvVars: []string{"LOTUS_ALLOW_SCHEMA_MIGRATION"},
```

Comment:
nit: `VISOR_ALLOW_SCHEMA_MIGRATION` seems more appropriate, but this isn't the first time we have been inconsistent with env var names

Reply:
Yeah, I keep swapping the lotus and visor names accidentally